test/Serilog.Sinks.XUnit.Tests/MessageSinkExtensionsTests.cs

```diff
@@ -0,0 +1,120 @@
+﻿using Xunit.Sdk;
```

Comment:
This one is out of place, and there's one in `SampleTests.cs` as well. Sorry to be nitpicky. I don't necessarily care if they're inside or out; just that they're all together.

Reply:
Not at all. I'm quite fussy myself and try to follow the coding style of what I'm changing. Sorry about that.